src/components/RecordLayout.js

```diff
@@ -3,7 +3,7 @@ import React from 'react'
 import Helmet from 'react-helmet'
 
 import { Route } from 'react-router-dom'
-import DrawViewContainer from './draw/DrawViewContainer'
+import DrawViewContainer from './recording/DrawViewContainer'
 
 export default ({
   children
```

Comment:
It is actually on L14 but do you want that to still be the DrawViewContainer or RecordingViewContainer?